### DISCUSSION TITLE
"Enhance CMD-A" doesn't work with TODOs

### DISCUSSION THREAD

**Discussion by cansar** [vslinko/obsidian-outliner#262] [Archive]
"Enhance CMD-A" is one of my favorite parts of Outliner, it's incredible useful when copy/pasting with lists. My lists will often also include TODOs for next steps to take on a current workflow. Unfortunately the Enhance CMD-A feature doesn't recognize those and copies the whole line. Copying it then leads to the following behavior. It would be wonderful to have these behave the same way as regular bullets.

<img width="294" alt="image" src="https://user-images.githubusercontent.com/92989/151412551-5376f39e-eb68-4ce8-b354-babfcaace9da.png">

**Comment by vslinko** [comment]
duplicate of https://github.com/vslinko/obsidian-outliner/discussions/221
